modules/standard/Random.chpl

```diff
@@ -2347,6 +2380,30 @@ module Random {
         }
       }
 
+      /*
+        Constructs a new stream of random numbers using the specified seed
+        and parallel safety.  Halts if provided an even seed.
+
+        :arg eltType: The element type to be generated.
+        :type eltType: `type`
+
+        :arg seed: The seed to use for the PRNG.  Defaults to `oddCurrentTime` from
+         :type:`RandomSupport.SeedGenerator`.
+        :type seed: `int(64)`
+
+        :arg parSafe: The parallel safety setting.  Defaults to `true`.
+        :type parSafe: `bool`
+
+        :returns: an owned RandomStream
```

Comment:
I don't understand why we need `type create` when a regular `new` will do.

Isn't `NPBRandomStream.create(int, seed)` redundant with `new owned NPBRandomStream(int, seed)` ?